Code/Framework/AzCore/AzCore/O3DEKernelConfiguration.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
```

Comment:
The `#pragma once` is at the wrong location within the file.

It should be after the copyright.